hooks/useIsUnsupportedChain.ts

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from 'react'
+import { useNetwork } from 'wagmi'
+
+interface Chain {
+  name: string
+  chainId: number
+}
+
+const TESTNET_CHAINS: Chain[] = [
```

Comment:
We could also just map out all of our supported chains, since they don't change often.
```
MAINNET_SUPPORTED_CHAINS = []
TESTNET_SUPPORTED_CHAINS = []
```

Reply:
This is probably fine as there's fewer testnet chains than mainnet chains.

Reply:
I think that viem has a property called `isTestnet` or something along those lines for their chains though.

Reply:
I saw that but It didn't seem to actually be returning the correct value. 
Ill take a look at it again.